src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Client/Data/Team/WorkItemStateColor.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Uno.AzureDevOps.Client
+{
+	public class WorkItemStateColor
```

Comment:
I'd simply name as 'WorkItemState'

Reply:
I asked myself the same but for consistence with API, I named it like it (one day we might need the exact serialized name :) )